src/i18n/lang/en.json

```diff
@@ -1,865 +1,866 @@
+  },
+  "DashboardProjectList": {
+    "title": "Recent projects"
+  },
+  "DashboardSpaceList": {
+    "title": "Recent spaces"
+  },
+  "DashboardWelcomeTile": {
+    "title": "Dashboard",
+    "message": "Hi {name}, it's great to see you again !"
+  },
+  "FileActionsCell": {
+    "openViewerButtonText": "Open in viewer",
+    "addTagsButtonText": "Add tags",
+    "addVersionButtonText": "Add version",
+    "manageAccessButtonText": "Manage access",
+    "createModelButtonText": "Define as model",
+    "removeModelButtonText": "Remove from models",
+    "versioningButtonText": "Add a version",
+    "visaButtonText": "Validation request"
+  },
+  "FilesActionBar": {
+    "moveButtonText": "Move to"
+  },
+  "FilesDeleteModal": {
+    "title": "Delete files",
+    "message": "This action will permanently delete the following files/folders :",
+    "messageVersion": "You are about to permanently delete the following files/folders and their associated version history:"
+  },
+  "FilesManager": {
+    "title": "Project Files",
+    "createModelNotification": "Model successfully created",
+    "structureImport": "Import a GED structure",
+    "gedDownload": "Download the GED",
+    "folderImport": "Import a folder"
+  },
+  "FilesManagerOnboarding": {
+    "text": "Upload your first file",
+    "uploadFileButtonText": "Upload a file",
+    "createFolderButtonText": "Create a folder",
+    "GEDStructureImport": "Import a GED structure",
+    "folderImport": "Import a file"
+  },
+  "FilesTable": {
+    "headers": {
+      "tags": "Tags"
+    }
+  },
+  "FileUploadButton": {
+    "addFileButtonText": "Upload a file"
+  },
+  "FileUploader": {
+    "uploadAreaText": "Drag and drop to store your project assets",
+    "uploadButtonText": "Select a model file"
+  },
+  "FolderAccessManager": {
+    "title": "Manage access rights",
+    "rightsHeader": "Rights",
+    "membersHeader": "Members",
+    "groupHeader": "Group",
+    "propagate": "Apply rights to sub folders (recursively)"
+  },
+  "FolderSelector": {
+    "selectedFolder": "Selected folder",
+    "moveHere": "Move here"
+  },
+  "GroupCardActionMenu": {
+    "changeColorButtonText": "Change color"
+  },
+  "GroupCardDeleteGuard": {
+    "title": "Delete group",
+    "message": "You are about to (definitely) delete group {name}, are you sure ?"
+  },
+  "GroupCardUpdateForm": {
+    "title": "Rename group"
+  },
+  "GroupCreationCard": {
+    "text": "Create a group",
+    "title": "New group"
+  },
+  "GroupMemberSelectionCard": {
+    "text": "Add a member"
+  },
+  "InvitationCard": {
+    "pendingMessage": "Invited - Pending.",
+    "resendLinkText": "Send invitation again ?",
+    "resendSuccessNotifText": "Invitation sent again",
+    "cancelSuccessNotifText": "Invitation canceled"
+  },
+  "InvitationForm": {
+    "emailInputPlaceholder": "Mail address",
+    "emailInputErrorMessage": "Invalid email",
+    "userAlreadyExistInputErrorMessage": "The user is already in your project",
```

Comment:
I don't know why but it looks like you do not have the latest version of this file...
I recently modified this file to optimize translation keys.
Do not not add this english translation for now to avoid conflicts, we will add it in POEditor instead.

Reply:
The modification of translation keys is perhaps on a branch not merged yet ?